_docs-as-code.rst

```diff
@@ -4,55 +4,170 @@ Documentation as code
+     lsst.afw.display/
+       index.rst
+       ..
+     lsst.afw.fits/
+       index.rst
+       ..
+     lsst.afw.geom/
+       index.rst
+       ..
+     lsst.afw.gpu/
+       index.rst
+       ..
+     lsst.afw.image/
+       index.rst
+       ..
+     lsst.afw.math/
+       index.rst
+       ..
+     lsst.afw.table/
+       index.rst
+       ..
+
+Package-oriented documentation is contained in a directory named after the package/Git repository itself.
+For ``afw``, this is the ``doc/afw/`` directory.
+
+Each module's documentation is contained in a directory named after the Python namespace of the module itself.
+For example, ``doc/lsst.afw.cameraGeom``.
+
+The ``_static/afw/`` directory hosts static files for the package's documentation.
+In Sphinx, "static" files are directly copied to the output built without intermediate processing.
+These could be PDFs or tarball downloads.
+This static content is stored in a ``_static/`` directory.
+So that static content from all packages can be integrated, each package must store static content in a sub-directory of the ``_static`` directory, such as ``_static/afw``.
+
+Each package also has ``doc/conf.py`` and ``doc/index.rst`` files, these facilitate :ref:`single-package development builds <per-package-builds>`.
+
+Finally, the ``doc/manifest.yaml`` file facilitates integrated documentation builds, as described in the :ref:`next section <integrated-build>`.
 
 .. note::
 
    The ``doc/`` directory was already used by the previous Doxygen-based documentation build system.
    However, during the transition from Doxygen to Sphinx-based builds, we do not expect any conflicts since content for the two system reside in non-overlapping files (``.dox`` versus ``.rst`` files for Doxygen and Sphinx, respectively).
    It should be possible to continue to build a Doxygen version of the documentation while the new Sphinx site is being prepared.
 
+.. _integrated-build:
+
+Integrated documentation: linking package documenation into the pipelines_lsst_io repository
+--------------------------------------------------------------------------------------------
+
+When pipelines_lsst_io_ is built, the package, module, and ``_static`` documentation directories of each package are linked into the cloned pipelines_lsst_io_ repository::
+
+   pipelines_lsst_io/
+      index.rst
+      ..
+      modules/
+        lsst.afw.cameraGeom/ -> link to /afw/doc/lsst.afw.cameraGeom/
+        ..
+      packages/
+        afw/ -> link to /afw/doc/afw/
+        ..
+      _static
+        afw/ -> link to /afw/doc/_static/afw
+        ..
+
+Module documentation directories are symlinked into pipelines_lsst_io_\ ’s ``modules/`` directory.
+Likewise, package documentation directories are symlinked into pipelines_lsst_io_\ ’s ``packages/`` directory. With all documentation content directories linked  into the pipelines_lsst_io_ directory, Sphinx is able to build the LSST Science Pipelines documentation if it were a unified project.
```

Comment:
Extra space between linked and into.

documentation if --> documentation as if